com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -478,7 +586,7 @@ private bool OnSceneEventProgressCompleted(SceneEventProgress sceneEventProgress
             {
                 using var nonNullContext = (InternalCommandContext)context;
                 ClientSynchEventData.SceneEventGuid = sceneEventProgress.Guid;
-                ClientSynchEventData.SceneIndex = SceneNameToIndex[sceneEventProgress.SceneName];
+                ClientSynchEventData.SceneIndex = GetBuildIndexFromSceneName(sceneEventProgress.SceneName);
```

Comment:
We don't need to check if this fails I assum?  We are 100% sure if we get here that `sceneEventProgress.SceneName` is valid?  Then I also wonder, why wouldn't we want `sceneEventProgress` to carry the `sceneIndex` vs. the scene string.

Reply:
Haha... dang it!  You are right!  This is a legacy design pattern that was back when the registered scenes indices values were different than the scenes in build list indices values.  We can just use the scene index value there for sure and extrapolate the scene name when it is done.  Nice catch!